cli/command/stack/remove.go

```diff
@@ -39,7 +39,17 @@ func runRemove(dockerCli command.Cli, opts removeOptions) error {
 	client := dockerCli.Client()
 	ctx := context.Background()
 
-	var errs []string
+	var (
+		errs    []string
+		secrets []swarm.Secret
+		configs []swarm.Config
+	)
+
+	version, err := client.ServerVersion(ctx)
```

Comment:
I missed this in the earlier PR. I think we should already have version stored, we shouldn't need to make another API call.

It's available from `cli.Client().ClientVersion()`. The name is misleading, but this version was already set from the server version.